corehq/apps/events/models.py

```diff
@@ -109,6 +118,24 @@ class Meta:
             models.Index(fields=("manager_id",)),
         )
 
+    def get_case_sharing_group(self, user_id):
```

Comment:
I think we need to put the fact that it's fake in this name too :)

Reply:
That is probably a good idea, and would make things clearer to other devs going through the code. I have faked the name for this as well. :smile: